Wrappers/Python/CMakeLists.txt

```
@@ -1,50 +1,32 @@
-option (BUILD_PYTHON_WRAPPER "Build Python Wrapper" ON)
+find_package(Python COMPONENTS Interpreter REQUIRED)
 
-if (BUILD_PYTHON_WRAPPER)
-  find_package(PythonInterp REQUIRED)
-
-  #set(PYTHON_DEST_DIR "" CACHE PATH "Directory of the Python wrappers")
-  if (PYTHON_DEST_DIR)
-    file(TO_CMAKE_PATH "${PYTHON_DEST_DIR}" TMP)
-    set(PYTHON_DEST "${TMP}")
-    unset(TMP)
-  endif()
-
-  message("CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS}")
-  if(NOT DEFINED)
-    set(CMAKE_BUILD_TYPE "RelWithDebInfo")
-  endif()
+if (PYTHON_DEST_DIR)
+  file(TO_CMAKE_PATH "${PYTHON_DEST_DIR}" TMP)
+  set(PYTHON_DEST "${TMP}")
```

is `TMP` really required?

```suggestion
  file(TO_CMAKE_PATH "${PYTHON_DEST_DIR}" PYTHON_DEST)
```

---

looks good